lib/interactor.rb

```diff
@@ -163,4 +163,30 @@ def call
   # Returns nothing.
   def rollback
   end
+
+  private
+
+  # Internal: Determine what arguments (if any) should be passed to the "call"
+  # instance method when invoking an Interactor. The "call" instance method may
+  # accept any combination of positional and keyword arguments. This method
+  # will extract values from the context in order to populate those arguments
+  # based on their names.
+  #
+  # Returns an Array of arguments to be applied as an argument list.
+  def arguments_for_call # rubocop:disable Metrics/MethodLength
```

Comment:
I removed positional argument support. I think it just muddied the waters and could be achieved just as well with keyword arguments.

As for the last two code examples in your comment above, I don't feel like the arguments accepted by the `call` _class_ method needs to necessarily agree 100% with the arguments accepted by the `call` _instance_ method, but scrapping positional argument support certainly does make it more consistent between the two, which I like. What are your thoughts, @hedgesky?